tap_salesforce/client.py

```diff
@@ -85,38 +91,48 @@ def __init__(
 
         self._login()
 
-    def get_tables(self) -> Generator[Tuple[Table, List[str], str], None, None]:
+    def get_tables(self, advanced_features_enabled=False) -> Generator[Table]:
```

Comment:
we also need to provide send_type and return type
```suggestion
    def get_tables(self, advanced_features_enabled=False) -> Generator[Table, None, None]:
```